lib/asset_sync/config.rb

```diff
@@ -94,7 +94,7 @@ def fail_silently?
     end
 
     def log_silently?
-      ENV['RAILS_GROUPS'] == 'assets' || self.log_silently == false
```

Comment:
Maybe if we warned on nil elsewhere in handling the config options? For something as benign as logging, casting `nil` to `false` is probably OK :)